### PR TITLE
Test2 v0 Installation Doc

### LINKED ISSUE #252: Add instructions for installing Test2::Suite
`cpm install -g Test2::Suite`, or `cpm install Test2::Suite` if local installation is necessary.
`libtest2-suite-perl` package for system Perl on Ubuntu/Debian.
`perl-test2-suite` package for system Perl on Arch.

### PR DESCRIPTION
**Pull request by rfilipo**

### CODE REVIEW

**Comment by mienaikage** [comment]
Apologies for the delay in looking at this, I'm currently on vacation and just finished a very long drive!

File: docs/INSTALLATION.md [hunk 1]
```diff
@@ -11,12 +11,105 @@ of Perl 5, take a look at 'Other Options'.
...
+
+Otherwise use one of the options bellow:
+
 ### Other Options:
 * [perlbrew](https://perlbrew.pl/): A tool to manage multiple perl installations
   in your `$HOME` directory. They are completely isolated perl universes.
 * [plenv](https://github.com/tokuhirom/plenv): Installs perls under your home
   directory and lets you install modules locally, and allows you to switch to
   arbitrary perl versions on your shell.
+* [App::cpm](https://metacpan.org/pod/App::cpm::Tutorial): cpm is yet another CPAN client (like cpan, cpanp, and cpanm), which is fast!
```
[the remark this answers is not in the document]
Comment (rfilipo) [reply docs/INSTALLATION.md]: Sure. I will do in the next chance. Thank you.

File: docs/INSTALLATION.md [hunk 2]
```diff
@@ -11,12 +11,105 @@ of Perl 5, take a look at 'Other Options'.
...
+sudo dnf groupinstall "Development Tools"
+
+```
+
+#### Debian, Ubuntu and similar
+
+```shell
+
+sudo apt update
+sudo apt install build-essentials
```
[the remark this answers is not in the document]
Comment (rfilipo) [reply docs/INSTALLATION.md]: Yes, it's a typo. Fixing.

File: docs/INSTALLATION.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (rfilipo) [reply docs/INSTALLATION.md]: Done. Closes #252

File: docs/INSTALLATION.md [hunk 3]
```diff
@@ -11,7 +11,79 @@ of Perl 5, take a look at 'Other Options'.
...
+echo 'export MANPATH=$HOME/perl5/man:$MANPATH' >> ~/.profile
+cpanm install Test2::Suite 
+
+```
+
+Optionally and if Perl 5 is up and running in your system just use 
+the command bellow to install the Test2 Suite globally:
+
+```shell
+
```
Comment (kotp) [inline docs/INSTALLATION.md]: Extra blank lines here.  I don't think they are necessary.

File: docs/INSTALLATION.md [hunk 4]
```diff
@@ -20,3 +92,57 @@ Use the `yum install perl-core` command to install them.
...
+* [CPANPLUS](https://metacpan.org/pod/CPANPLUS): The CPANPLUS library is an API to the CPAN mirrors and a collection of interactive shells, commandline programs, etc, that use this API.
+* [App::cpm](https://metacpan.org/pod/App::cpm::Tutorial): cpm is yet another CPAN client (like cpan, cpanp, and cpanm), which is fast!
+
+---
+
+### Install Test2::V0 under alternative environments
+
+0. Using cpm
+```shell
+
```
Comment (kotp) [inline docs/INSTALLATION.md]: Here as well... matching one below the code line.

File: docs/INSTALLATION.md [hunk 5]
```diff
@@ -11,7 +11,79 @@ of Perl 5, take a look at 'Other Options'.
...
+The Perl 5 track in Exercism uses [Test2 Suite](https://metacpan.org/pod/Test2::Suite) 
+to verify the codes. It's mandatory installing this suite to have your tests verified.
+
+The last bundle is _Test2::V0_.
+
+We strongly recommend you installing local::lib and cpanminus, 
+so you could install packages and work under your own workspace. 
+
+```shell
+
```
Comment (kotp) [inline docs/INSTALLATION.md]: Extra blank lines around the commands to invoke.

Comment (rfilipo) [reply docs/INSTALLATION.md]: Done. All unnecessary blank lines were removed. Thank you, Kotp for your attention! :+1:

**Comment by rfilipo** [comment]
I propose to split this PR in more simple ones. It's better a PR dealing with only one question. 
We have too many here, at least:
1. Installing Perl.
   - Global
   - Local
2. Installing a package manager
   - Explain options
   - Global
   - Local
3. Installing Test2::V0
   - What exercises need it
   - Global
   - Local

Another question is if we really need Test2::Suite to test all exercises or only some.
We must discuss it before any changes in the documentation, please.

**Pull request closed without merging**